### ISSUE TITLE
incorporating depreciation into the model

### ISSUE THREAD

**Issue by ktegan** [S120/PKSFC#9]
I am trying to understand SFC modeling and the PKSFC project looks great.  I have a question and would appreciate any help you can provide.

The SFC modeling approache prides itself on making sure that all money flows are recorded and net out to zero: one person's income is another person's payment.  However, it seems to me that the depreciation or appreciation of value of goods should be able to be modeled without netting out to zero and I would love to know if this is possible to model and if so how so.

Let me try to explain with an example.  I would like to take a simple PKSFC model and look at the results for each of the following fiscal policies:
 - the government increases annual spending by $100B and uses this money to pay people to dig and fill ditches (output which the market values at $0)
 - the government increases annual spending by $100B and uses this money to pay people to grow food (output which the market values at $100B)

In many cases it might be OK to assume that $100B of income produces $100B of goods or services (especially in the private sector) but I'm interested in this extreme example.  (1) do you agree that the above two scenarios may have different rates of inflation since payments are identical but the amount of goods to purchase are not?  And (2) is there any way to model this within the PKSFC framework?  One idea I had would be to make a new stock of "consumables" which would be the market value of goods produced and would tie together the market value of business output (which may be different from what was paid), consumption and inflation.  To keep all flows netted to zero you could make a fictitious ledger that gets income when the market value of a consumable value depreciates and gives payment when a consumable appreciates in value.

I hope that made some amount of sense.  Thanks!

**Comment by antoinegodin** [comment]
I think you are somehow confusing two concepts: 
- One is of depreciation of goods: i.e. the fact that some goods are durable but they stop working properly with time, which leads to a financial amortization (from an accounting perspective the value of that good reduces with time). Financial amortization which might be different from the actual depreciation.
- The second is the fact that you might produce things that do not have any market value. Your example is actually exactly that, in both case output is produced (say 200k ditches have been dug and filled, or 200T of vegetables have been grown) but the market value of ditches is 0 (i.e. its price is 0) while the market value of vegetable is $500 per ton. 

In your case I would simply have two goods: one which is marketable with a positive price and one which is not marketable with a price=0. The wages would nonetheless be paid but the output has no value, hence leading to deficit in that sector: profits = output*price - wages*employment which in your case would be: -$100B=200k*0-$100B. Stock-Flow Consistency is respected as long as that deficit is matched by an equivalent inflow (in your case it was done via the governmental transfer but you could also think of a raise in debt, provided you find someone willing to do so).

**Comment by ktegan** [comment]
Yes, indeed I was thinking of the second case.  I have a follow up question below so I will hold off on closing this right now but feel free to close this issue.

I am hoping to come up with a simple model that uses endogenous money (money supply driven by demand for credit) as well as differentiating GDP calculated by income (payments same for ditches and vegetables) and output (vegetables have value while ditches do not) and using the GDP output value for inflation calculations.  If you know of an existing model off the top of your head that does this please let me know.

Thanks for the help, it is much appreciated!

**Comment by antoinegodin** [comment]
I'm not sure I understand you. Say you have a very simple economy composed of two sectors (Ditches and Vegetables), both of them producing without capital, only labour, and there is no governement and we assume that banks finance everyone. Production side of GDP is as you indicated: D x pD + V x pV=V x pV, income side of GDP is the sum of wages and profits. Wages=wD+wV, profits is sales-costs by sector so for V: pV x V-wV = FV, while for D: 0-wD = FD. So the GDP from the income side is wV+wD + FV + FD = V x pV. You would not be able to deflate one by the other to compute inflation. 

Summing up D and V would make no sense because you are summing up ditches and vegetables which do not have the same unit. Technically what you should do is compute a nominal GDP and deflate it by some sort of price index to obtain a real GDP (which is a very abstract concept). In our little example however it would not work because if a price is zero, the goods produced do not enter in nominal GDP and hence you cannot deflate them.

**Comment by ktegan** [comment]
I think this is showing my novice level of understanding of the SFC models because I didn't realize that the income GDP calculations included profits.  I think your point here is that in your simple example GDP is the same whether it is computed using output or income.  It sounds like my question is a little nonsensical and I need to dive into the Godley and Lavoie book when I get my hands on it shortly.

Two very short questions if you have time (and obviously feel free to close this issue):
 - in the pksfc tutorials if I wanted to modify the marketable output (did we make ditches or vegetables?) for either the private sector or government (whatever is easier) what variable would I change?
 - Is there a reason why inflation seems to generally be calculated using its own heuristic like the GDP output gap (Fiscal Policy in a Stock-Flow Consistent Model) or as a markup influenced by the power of firms and employees (A Post-Keynesian Stock-Flow Consistent Macroeconomic Growth Model) instead of a more direct equation involving changes in the broad money supply and changes in GDP?  It seems like many models include credit demand and various relationships consistent with the idea of endogenous money, but these values related to money supply do not feed into the calculations for inflation.  Maybe these different approaches are totally consistent, maybe money supply to GDP ratios are unstable, or maybe I'm confusing cause and effect.  Again, I may be out of my element here but that strikes me as odd.

Thank you for your help and your patience!

**Comment by antoinegodin** [comment]
In most models, output is actually generated in the private sector. You would need to change the structure of the model to be able to incorporate governmental production. It is thus more than a simple variable to change.

I don’t think it makes too much sense to want to connect inflation directly to the quantity of money circulating in the economy. You will need to explain what is the mechanism by with prices are revised upwards: is it the case that firms are increasing their mark-up or is it the case that their unit cost are increasing (that is assuming mark-up pricing)? What I’m trying to say is that you need to justify your inflation by economic theory which is consistant with the SFC framework.
